Firestore/src/Query.php

```diff
@@ -355,7 +355,11 @@ public function where($fieldPath, $operator, $value)
         }
 
         if ($escapedPathString === self::DOCUMENT_ID) {
-            $value = $this->createDocumentReference($basePath, $value);
+            $value = $operator === FieldFilterOperator::IN
```

Comment:
This logic is a bit much for a ternary. Let's move it to an if/else.

Reply:
Expanded it